docs/src/quickstart/hello_klio_batch.rst

```diff
@@ -0,0 +1,127 @@
+.. _hello-klio-batch:
+
+Hello Klio Batch Example
+========================
+
+This guide will show you how to set up your development environment to implement Klio,
+create an example Klio batch job and run it on DirectRunner. If you are interested in building
+a streaming Klio job then checkout the :ref:`Klio Streaming Quickstart Guide<hello-klio-streaming>`.
+
+.. attention::
+
+    Be sure to follow the :doc:`installation instructions <installation>` before continuing on.
+
+
+Create a New Klio Batch Job
+---------------------------
+
+:violetemph:`First`, initialize the ``klio_quickstart`` project directory for ``git``:
+
+.. code-block::
+
+    $ git init
+
+.. caution::
+
+    If the |klio-cli|_ was installed via :ref:`option 2 <install-option-2>` or :ref:`option 3 <install-option-3>`, make sure you're ``klio-cli`` virtualenv is activated.
+
+
+:violetemph:`Next`, within your project directory, run the following command:
+
+.. code-block:: sh
+
+    $ klio job create --job-name klio-quick-start --job-type batch --create-resources --use-defaults
```

Comment:
Since this is a long line that most likely ends up needing some scrolling, so folks may not see the last options. Maybe break it up like so:

```rst
.. code-block:: sh

    $ klio job create \
      --job-name klio-quick-start \
      --job-type batch \
      --create-resources \
      --use-defaults
```